Resources/Server Info/Intro.txt

```diff
@@ -1 +1,8 @@
-Hi and welcome to Space Station 14! This tutorial will assume that you know a bit about how SS13 plays. It's mostly intended to lay out the differences from SS13.
\ No newline at end of file
+Welcome to space station 14! You have just started your shift aboard a Nanotrasen space station and along with your crewmates should cooperate, perform your job role to the best of your ability and have fun (not at the expense of others hopefully).
+
+Please make sure you understand the server rules (F1) and if you have any questions, consult this guide, your colleagues on shift in person or over radio or via discord. You can also ask admins for help (ahelp) by hitting the escape key > admin help and typing a message to any online admins. Especially do this if you witness someone breaking rules.
+
+To reiterate, you are all a crew of people who work together (even the clown) and you should not be harming each other unless you have an extremely good reason (self defence) and even then, call security over radio, stun and detain, do not commit murder as a non antagonist (see antagonist further down).
+
+
+Instead of intents, we have combat mode. Check controls above for its keybind. You can't attack anybody with it off, so no more hitting yourself with your own crowbar.
```

Comment:
Should say controls BELOW

Reply:
Thanks, copied that bit from the old guide

Reply:
Maybe add something like "Or look at the slots on the left" (My English grammar is terrible, but you get the idea)